azurerm/resource_arm_action_group_test.go

```diff
@@ -0,0 +1,242 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMActionGroup_basic(t *testing.T) {
+	resourceName := "azurerm_action_group.test"
+	ri := acctest.RandInt()
+	config := testAccAzureRMActionGroup_basic(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMActionGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMActionGroupExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "enabled", "true"),
+					resource.TestCheckResourceAttr(resourceName, "email_receiver.#", "2"),
+					resource.TestCheckResourceAttr(resourceName, "email_receiver.0.email_address", "admin@contoso.com"),
+					resource.TestCheckResourceAttr(resourceName, "email_receiver.1.email_address", "devops@contoso.com"),
+					resource.TestCheckResourceAttr(resourceName, "sms_receiver.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "sms_receiver.0.country_code", "1"),
+					resource.TestCheckResourceAttr(resourceName, "sms_receiver.0.phone_number", "1231231234"),
+					resource.TestCheckResourceAttr(resourceName, "webhook_receiver.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "webhook_receiver.0.service_uri", "http://example.com/alert"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAzureRMActionGroup_empty(t *testing.T) {
```

Comment:
we should update this to use the suffix `basic` since that follows the convention of our other tests with only the required/basic structure